eng/Packages.Data.props

```diff
@@ -58,8 +58,8 @@
   <ItemGroup Condition="'$(IsClientLibrary)' == 'true'">
 
     <!-- BCL packages -->
-    <PackageReference Update="System.Buffers" Version="4.5.0" />
-    <PackageReference Update="System.Memory" Version="4.5.3" />
+    <PackageReference Update="System.Buffers" Version="4.5.1" />
+    <PackageReference Update="System.Memory" Version="4.5.4" />
```

Comment:
I'm so confused as to why it shows any changes in this file.

Reply:
That is weird.

Reply:
Ah. I hadn't resolved  conflicts.